modules/dxp/apps/portal-workflow/portal-workflow-metrics-test/src/testFunctional/tests/WorkflowMetrics.testcase

```diff
@@ -960,6 +960,7 @@ definition {
 	@description = "LRQA-53660 Automate LPS-87571 [Reports] Filter the list of items for a certain process"
 	@priority = "5"
 	test FilterAllItemsList {
+		property portal.upstream = "quarantine";
```

Comment:
Hey @sirfabios I think that the property for P5 tests is `property portal.acceptance = "quarantine";`

Reply:
> Hey @sirfabios I think that the property for P5 tests is `property portal.acceptance = "quarantine";`

Done